dashboard/app/views/layouts/_footer.html.haml

```diff
@@ -37,3 +37,18 @@
           %br/
           %br/
           !="&copy; Code.org, #{Time.now.year}"
+
+- if DCDO.get('hoc_secret', nil)
+  // Set secret
+  - # If enabled, set a client-side secret that allows admins to access the HOC 2015 levels.
+  - # This is a temporary measure that will be removed shortly.
+  - if current_user && current_user.admin?
+    :javascript
+      //<![CDATA[
+      localStorage.setItem('hoc_secret', '#{DCDO.get("hoc_secret", nil)}');
+      //]]>
+  - else
+    :javascript
+      //<![CDATA[
+      localStorage.removeItem('hoc_secret');
```

Comment:
Wouldn't this mean that visiting any cached page would remove the secret from localStorage?